.github/workflows/linux.yml

```diff
@@ -39,16 +40,23 @@ jobs:
 
         - ssl: ssl_ON
           INSTALL_SSL: libssl-dev
-          EXTRA_CMAKE_FLAGS: -DWITH_OPENSSL=ON
+          OPENSSL_CMAKE_OPTION: -DWITH_OPENSSL=ON
 
         - ssl: ssl_OFF
-          EXTRA_CMAKE_FLAGS: -DWITH_OPENSSL=OFF
+          OPENSSL_CMAKE_OPTION: -DWITH_OPENSSL=OFF
+
+        - dependencies: dependencies_SYSTEM
+          DEPENDENCIES_CMAKE_OPTIONS: -DWITH_SYSTEM_ABSEIL=ON -DWITH_SYSTEM_LZ4=ON -DWITH_SYSTEM_CITYHASH=ON
+          INSTALL_DEPENDENCIES: libabsl-dev liblz4-dev libcityhash-dev
+
+        - dependencies: dependencies_BUILT_IN
+          DEPENDENCIES_CMAKE_OPTIONS: -DWITH_SYSTEM_ABSEIL=OFF -DWITH_SYSTEM_LZ4=OFF -DWITH_SYSTEM_CITYHASH=OFF
 
     steps:
     - uses: actions/checkout@v2
 
     - name: Install dependencies
-      run: sudo apt-get install -y cmake ${{ matrix.INSTALL }} ${{ matrix.INSTALL_SSL }}
+      run: sudo apt-get install -y cmake ${{ matrix.INSTALL }} ${{ matrix.INSTALL_SSL }} ${{ matrix.INSTALL_DEPENDENCIES }}
 
     - name: Configure CMake
       run: |
```

Comment:
There seems to be a glitch in this multi-line value, maybe consider using `run: >-` instead of `run: |`. Not sure why it worked before...